data/download_data.py

```diff
@@ -302,7 +302,7 @@ def download_file(s3_url: str, out_dir: str, file_name: str):
     print('Saved to ', local_file)
 
 
-def download(resource_key: str, out_dir: str = None):
+def download(resource_key: str, out_dir: str = None, keep_gzip: bool = False):
```

Comment:
hmm...
Why not to create a copy of resource downloading configuration with compressed set to False?
I think it is a better option with no functional code changes & bloating 'download' method with extra parameters

Reply:
Hmm, I am not sure if I understand correctly, but if there is another resource, wouldn't it download both ".tsv.gz" and ".tsv" when people use prefix?

Reply:
Just name them with different resource keys.